src/sentry/static/sentry/app/views/organizationDashboard/index.jsx

```diff
@@ -1,21 +1,91 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import createReactClass from 'create-react-class';
+import {Flex, Box} from 'grid-emotion';
+import styled from 'react-emotion';
 
 import OrganizationState from '../../mixins/organizationState';
 import OldDashboard from './oldDashboard';
 import ProjectNav from './projectNav';
+import TeamMembers from './teamMembers';
+import ProjectCard from './projectCard';
+import getProjectsByTeams from '../../utils/getProjectsByTeams';
+import withTeams from '../../utils/withTeams';
+import withProjects from '../../utils/withProjects';
 
 class Dashboard extends React.Component {
+  static propTypes = {
+    teams: PropTypes.array,
+    projects: PropTypes.array,
+  };
+
+  componentWillMount() {
+    $(document.body).addClass('org-dashboard');
+  }
+  componentWillUnmount() {
+    $(document.body).removeClass('org-dashboard');
+  }
+
   render() {
+    const {projects, teams} = this.props;
+    const {projectsByTeam} = getProjectsByTeams(teams, projects);
+    const projectKeys = Object.keys(projectsByTeam);
+
     return (
       <div>
         <ProjectNav />
-        New dashboard placeholder
+        <div>
+          {projectKeys.map((slug, index) => {
+            return (
+              <TeamSection key={slug} renderBorder={index !== projectKeys.length - 1}>
```

Comment:
I think we should use `render`-prefixed props when we want to pass it a component to render. Maybe `showBorder` would be better here